src/popup/content_links.js

```diff
@@ -35,7 +35,10 @@ Popup.generateContentLinks = function(location, currentEnvironment, renderingApp
 
   if (renderingApplication == "smartanswers") {
     if (currentUrl.match(/\/y\/?.*$/)) {
+      linkToTemplateOnGithub = "javascript: if (typeof(linkToTemplatesOnGithub) === 'function') { linkToTemplatesOnGithub(); };"
```

Comment:
Should this give an indication to the user when it doesn't work? Maybe `alert()` when `linkToTemplatesOnGithub` doesn't exist?